functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import pypyodbc
+import os
+import azure.functions as func
+import json
+
+

```

Comment:
Would love to see some comments in this file. I think it would definitely be helpful in identifying what's going on at the different stages.

Reply:
Will add comments!